docs/FAQ.md

```diff
@@ -0,0 +1,127 @@
+# Frequently Asked Questions
+
+Links to sections:
+- [Common](#common)
+- [PyTorch](#pytorch)
+- [TensorFlow](#tensorflow)
+- [ONNX](#onnx)
+
+
+## Common
+
+### What is NNCF for?
+NNCF takes a deep learning network model object and modifies it for faster inference.
+Within NNCF, the process of modification is colloquially known as compression.
+
+Sometimes this is not possible to do without the loss of accuracy for the network without losing quality.
+NNCF provides algorithms that strive for minimal or zero loss of accuracy, which can be applied, depending on the algorithm, during training, fine-tuning or post-training.
+
+### Does the Neural Network *Compression* Framework provide *lossless compression*?
+Not in the way the term "lossless compression" usually appears in literature. 
+Under "compression" we mean the preparation of the model for *future* efficient execution of this model in the OpenVINO Inference Engine. 
+Under "future" we mean that the process of compression is usually an offline, one-time step before the model is being used in production, which provides a new model object that could then be used instead of the original to run faster and take up lower memory without significantly losing accuracy.
+
+No *compression* in the sense of archiving or entropy coding is being done during NNCF compression.
+
+### How does your compression make inference faster?
+General, well-known, literature-backed techniques of neural network inference acceleration are applied, with Intel HW/runtime specifics in mind.
```

Comment:
This is the only answer in this FAQ that I find vague. It goes too far to give a crash course in techniques here, but a link to the paper could be useful https://arxiv.org/abs/2002.08679 And maybe mention some techniques here, like "like quantization, filter pruning and knowledge distillation".

Reply:
Applied your suggestion pretty much verbatim, thank you! Also removed the FILLME.